src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './header.scss';
+
+export const Header: React.FC = () => {
+  const activeClassName = 'Header__is-active';
+
+  return (
+    <nav className="Header__navbar">
+      <NavLink
+        to="/"
+        className={({ isActive }) => (
+          isActive ? activeClassName : 'Header__link')}
+      >
+        Home
+      </NavLink>
+
+      <NavLink
+        to="/people"
+        className={({ isActive }) => (
+          isActive ? activeClassName : 'Header__link')}
+      >
```

Comment:
BTW, when you ckick to nav link it smart compnent self add class 'active', so you don't need add it, just use .active class in your app.scss file

Reply:
also you deployed page don't open, fix it
![image](https://user-images.githubusercontent.com/73961448/171107862-01e15dc7-452e-4bcb-9085-065a8a2441d4.png)